examples/compiler/01_lowering.ipynb

```diff
@@ -0,0 +1,363 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In Devito, an `Operator` carries out three fundamental tasks: generation of low-level code, JIT compilation, and execution. Overall, the lowering process performed by an `Operator` – from high-level equations to dynamically compiled and executable code – consists of multiple compiler passes, summarized in the Figure bellow. The following series of notebooks will expose those compilation stages."
```

Comment:
there is no figure below

have you checked that this premise is not redundant in the other notebooks ?

Reply:
please see comment bellow

Reply:
added link as pdf!

Reply:
i noticed the pdf was not being displayed.
i found [this](https://stackoverflow.com/questions/19470099/view-pdf-image-in-an-ipython-notebook) solution, but i'm not sure if i'm overkilling it.
would you have any advice on it?